.github/workflows/minor-schema-version-bump.yml

```diff
@@ -215,3 +215,6 @@ jobs:
           branch: auto/update-convert-py-to-${{ env.new_version }}
           base: main
           reviewers: bento007 # TODO assign curators
+          add-paths:
```

Comment:
It looks like this needs to be slightly different syntax (tried it in the demo branch and it failed)

See https://github.com/peter-evans/create-pull-request#add-specific-paths